pkg/models/repo_config.go

```diff
@@ -16,10 +16,21 @@ limitations under the License.
 
 package models
 
-// RepoConfig holds all repo configuration
+// Config struct represents the basic project setup values
+type Config struct {
```

Comment:
`Config` might be too generic of a name here since it'll be unique among all of our `pkg/models`.

Reply:
Hm.. Yeah. I changed it to `ProjectConfig`.